core/lib/zksync_core/src/api_server/web3/namespaces/eth.rs

```diff
@@ -613,6 +612,9 @@ impl EthNamespace {
         const METHOD_NAME: &str = "new_filter";
 
         let method_latency = API_METRICS.start_call(METHOD_NAME);
+        let Some(installed_filters) = &self.state.installed_filters else {
+            return Err(Web3Error::NotImplemented);
+        };
```

Comment:
Nit: You can use `ok_or` to convert an `Option` to a `Result` more idiomatically.